src/scripts/surf.ts

```diff
@@ -0,0 +1,74 @@
+// Description:
+//  Surf server stat comparison
+//
+// Commands:
+//  hubot surf me - compare surfers
+//
+// Author:
+//  Steve Shipsey
+
+import Axios, { AxiosResponse } from 'axios';
+import * as Cheerio from 'cheerio';
+import { Response, Robot } from 'hubot';
+import { table, TableUserConfig } from 'table';
+
+const baseUrl = 'https://snksrv.com/surfstats';
+const surfers = [
+  'STEAM_1:0:6030042',
+  'STEAM_1:0:1166568',
+];
+const headers = ['Surfer', 'Points', 'Map Completions'];
+
+const fetchSurfer = async (steamId: string) => await Axios.get<any>(`${baseUrl}`, {
+  params: {
+    id: steamId,
+    view: 'profile',
+  },
+});
+
+const parse = (cheerio: any): string[] => {
+  const $ = Cheerio.load(cheerio.data);
+  return [
```

Comment:
I think the return here should be a typed object. 

```
interface SurfData { ... }
```

or something

Reply:
Oh, nvm. It needs to be arrays because thats what table is expecting...

Reply:
Basically my main gripe is that the sortby relies on an arbitrary index instead of something like `.points`. this isn't production code so idc but it might be better to turn an object into table's expected format instead of just assuming that all the way through

Reply:
eh, its fine